frontend/public/components/utils/webhooks.tsx

```diff
@@ -58,15 +58,23 @@ export const WebhookTriggers: React.FC<WebhookTriggersProps> = (props) => {
   const [webhookSecrets, setWebhookSecrets] = React.useState<K8sResourceKind[]>([]);
   const [secretErrors, setSecretErrors] = React.useState<string[]>([]);
   const [isLoaded, setLoaded] = React.useState(false);
+  const [webhooks, setWebhooks] = React.useState<WebhookTrigger[]>([]);
+  React.useEffect(() => {
```

Comment:
We're looking at this separately in #3929. I'd probably just remove this from the PR

Reply:
Should I just disable the exhaustive deps rule until that PR merges then? This will have a lint failure and can't be merged otherwise.

Reply:
Or just block this PR until that merges?

Reply:
It's a warning not an error, so it won't block merge. (We should make it an error instead, but we can wait for #3929)